example/android/build.gradle

```diff
@@ -5,15 +5,14 @@ buildscript {
     }
 
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.2.1'
+        classpath 'com.android.tools.build:gradle:3.5.0'
```

Comment:
Should we bump it for the library as well?

Reply:
I think so because it's default for the current stable version of Flutter.

Reply:
Can you take care of that in this PR?